tenzir.yaml.example

```diff
@@ -7,6 +7,10 @@ tenzir:
   # The host and port to listen at and connect to.
   endpoint: "localhost:5158"
 
+  # The platform auth token for app.tenzir.com. This is an alternative to
+  # the dedicated platform.yaml file. The key has the following format:
+  #token: <tenant-id>:<api-key>[:<control-endpoint>]
```

Comment:
```suggestion
  #platform-token: <tenant-id>:<api-key>[:<control-endpoint>]
```

WDYT?

Reply:
I prefer the brevity of `TENZIR_TOKEN` over `TENZIR_PLATFORM_TOKEN` to be honest. But it's not a very strong opinion. Would be curious to hear what the @tenzir/platform-squad thinks first, since I kind of made this change without a lot of discussion.

Reply:
> But it's not a very strong opinion.

FWIW, me neither. It just felt more intuitive, and I wanted to share my first gut reaction.